apps/music/test/marionette/lib/playlisthelper.js

```diff
@@ -4,6 +4,10 @@
 
 var PlaylistHelper = {
 
+  mainTitle: function(element) {
+    return element.findElement('.list-main-title').text();
```

Comment:
http://mozilla-b2g.github.io/marionette-js-client/api-docs/files/lib_marionette_element.js.html#l168

That's what it returns. We use it elsewhere too.